nbs/97_test_utils.ipynb

```diff
@@ -1,23 +1,13 @@
 {
  "cells": [
-  {
```

Comment:
Careful not to delete cells like this in your PRs! :) I'll add it back after I merge

Reply:
why it happen? 
I never touched the ipynb's without jupyter...

Reply:
It means you deleted the cell in Jupyter.